### ISSUE TITLE
Support for outputting position files

### ISSUE THREAD

**Issue by felixeriksson** [BenMMcLean/kicad-gerber#1]
Hi!
Thank you for publishing this action. For me, it would be very useful to be able to export position files with this action. I wonder if you think it would be suitable to have support for that in this library.

If so, would it be acceptable that it always exports the position file? That, I believe, would simply be a matter of adding another line to `entrypoint.sh`.

**Comment by emilymclean** [comment]
I have added a new option "position-format", which defaults to none. If you put in "csv" "gerber" or "ascii" the workflow will generate a position file.

**Comment by felixeriksson** [comment]
Wow, thanks!! 🥇 

I tried it out, and it works great if I set `position-format: gerber`, but not with `csv` or `ascii`. The output from the job doesn't really indicate any error - the last line I get from this step is `Loading board`, which is the same that I get if I locally run `kicad-cli pcb export pos --format ascii -o production-files my-pcb.kicad_pcb` - but the job fails after that. If I instead use `csv` or `ascii`, I get 2 lines of `Loading board`, also as expected since `kicad-cli` is invoked 2 times in this case, but then the job proceeds successfully.
